public/static/js/init.py

```diff
@@ -8,10 +8,13 @@
 import json
 import re
 from collections import deque
-from pyodide import to_js
+from pyodide.ffi import to_js
 
 print(sys.version)
 
+class NotEnoughInputsError(Exception):
```

Comment:
random change to display a more useful message when the user asks for too many inputs. E.g. user has 2 inputs but we only provided 1.